specs/Normative-Types-Specification.rst

```diff
@@ -360,13 +361,13 @@ Suppose a non-terminal term has a definition of the form
 
 where:
 
-<*non-terminal*>
+``<*non-terminal*>``:
```

Comment:
How should this content be formatted?

Reply:
As far as I can tell (the original author of this text is Greg White), this is rendered correctly on RTD as it is.

Reply:
(meaning line 363.)

Reply:
Otherwise fine.